client/src/components/DragAndDrop/column.jsx

```diff
@@ -0,0 +1,42 @@
+import { Draggable } from 'react-beautiful-dnd';
+import ColumnList from './dNdComponents/itemList';
+import dndStyles from './assets/dndStyles';
+import { Box, Grid, Typography } from '@material-ui/core';
+
+export default function BoardColumn({ title, items, index, isScrollable, isCombineEnabled, boardID }) {
+  const classes = dndStyles();
+
+  return (
+    <Draggable draggableId={title} index={index}>
+      {(provided) => (
+        <Grid
+          className={classes.column}
+          container
+          item
+          xs={12}
+          sm={12}
+          ref={provided.innerRef}
+          {...provided.draggableProps}
+        >
+          <Grid item xs={12} sx={{ height: '50px' }} {...provided.dragHandleProps}>
+            <Typography className={classes.columnHeader} gutterBottom variant="h5" component="h5">
+              {title}{' '}
```

Comment:
You can simply use a template string here `{'${title} '}`
Please note that the single quote you see in my suggestion should be a backtick in your case, so you don't do a copy-paste. I used it because backtick has a special meaning in markdown.